internal/model/os/os.proto

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+syntax = "proto3";
+package f5.nginx.agent.internal.model.os;
+
+option go_package = "github.com/nginx/agent/v3/internal/model/os";
+
+message Process {
```

Comment:
`proto` files are typically contract definitions, are `os.proto` and `instances.proto` both not meant for usage outside the project?

Reply:
Moving proto files to `/api/grpc`